src/types/variabletimezone.jl

```diff
@@ -38,9 +38,4 @@ function Base.isequal(a::VariableTimeZone, b::VariableTimeZone)
     )
 end
 
-function Base.hash(tz::VariableTimeZone, h::UInt)
-    h = hash(tz.name, h)
-    h = hash(tz.transitions, h)
-    h = hash(tz.cutoff, h)
-    return h
-end
+Base.hash(tz::VariableTimeZone, h::UInt) = hash(tz.name, hash(VariableTimeZone, h))
```

Comment:
I'd like to keep the hashing function definitions similar thoughout TimeZones.jl:
```suggestion
function Base.hash(tz::VariableTimeZone, h::UInt)
    h = hash(:timezone, h)
    h = hash(tz.name, h)
    return h
end
```
I added in the `:timezone` seed to avoid `hash(::String) == hash(::VariableTimeZone)` (originally though of here https://github.com/JuliaTime/TimeZones.jl/pull/147#pullrequestreview-146301173) and I used a symbol as the seed  (discussed here: https://github.com/JuliaTime/TimeZones.jl/pull/147#discussion_r210170560)